lib/id3c/cli/command/etl/redcap_det.py

```diff
@@ -100,11 +88,21 @@ def decorator(routine: Callable[..., Optional[dict]]) -> click.Command:
             help        = "Write the output FHIR documents to stdout. You will likely want to redirect this to a file",
             default     = False)
 
+        # XXX FIXME: I don't think @command_for_project should even be
+        # providing the "cache" parameter.  What is cached and where it is stored is
+        # something specific to each REDCap DET routine, not a global invariant.
+        #   -trs, 19 Dec 2019
+        @click.option("--cache-file",
```

Comment:
Since this is now part of the user interface, let's give this a more descriptive name? Lots of good choices here. Maybe `--geocode-cache`?